style.css

```diff
@@ -0,0 +1,28 @@
+#title{
+    text-align: center;
+ }
+ 
+ 
+ #image{
+    max-width: 100% ;
+    height: auto;
+    display: block;
+ 
+
+ }
+ 
+ #img-divv{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+ 
+ }
+ #tribte-info{
+    text-align: center;
+ }
```

Comment:
Please make sure to give a correct ID because it will not give the intended style.
![Comment](https://github.com/Kccendana/code-review/assets/72602163/5fef4d40-be32-41d6-87c3-b173f41811ad)

Reply:
Good job! Your comments on the CSS are great. However, it would be helpful to focus on commenting on the HTML structure and elements instead of just the styles. 

For example, you could mention that 'the h2 title is well-implemented, but it doesn't match the desired design. It's important to address this aspect as well.' 

Keep up the good work